Kickstarter-iOS/Features/ProjectPage/Controller/ProjectPageViewControllerTests.swift

```diff
@@ -41,12 +41,12 @@ internal final class ProjectPageViewControllerTests: TestCase {
           caption: "camera"
         ),
         AudioVideoViewElement(
-          sourceURLString: "https://source.com",
+          sourceURLString: "https://not-a-real-url-dot-com",
           thumbnailURLString: nil,
           seekPosition: .zero
         ),
         ExternalSourceViewElement(
-          embeddedURLString: "https://source.com",
+          embeddedURLString: "https://not-a-real-url-dot-com",
           embeddedURLContentHeight: 123
         )
```

Comment:
I wonder if we should add a background or some local asset to this, so we can tell if it displays at all? I totally agree that there should not be a network fetch here, if that's what's happening. I'm just not sure if we're now testing `ExternalSourceViewElement` at all. If we're not, and we don't have a way to test it that doesn't involve networking, we should maybe just remove it entirely.

Reply:
Agreed! 

I got stuck trying to implement that, though. When I tried to remove the `ExternalSourceViewElement` entirely, I ran into different issues - namely that the play button for the video element above it stopped displaying, which also broke the tests. ([Here's the circleCI job](https://circleci-tasks-prod.s3.us-east-1.amazonaws.com/storage/artifacts/118e7113-ad83-4256-acf4-209b99df3884/120e3b3a-0e8b-440f-83a7-bfb40e7ee70a/0/Failed_Snapshot_Tests/ProjectPageViewControllerTests/testLoggedOut_NonBacker_LiveProjectSwitchedToCampaignTab_Success.lang_de_device_pad.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAQVFQINEOOVYZZ5WC%2F20231018%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20231018T183635Z&X-Amz-Expires=60&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEJP%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaCXVzLWVhc3QtMSJHMEUCICHI3Ygthlc5kq6yN7lWx3wQm9eKAAyIOWriApF97oSuAiEAr%2FUfClrwNTxA%2FbJZA0EDcwj6LLK08db6t%2BjrLE6YzegqtAIIrP%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FARADGgwwNDU0NjY4MDY1NTYiDGaYkA2aZoFAMGvdPiqIAksWZ45VztD9MLFpxMAcJt%2FJU1f2FwBiAsJNHMQHepHXKc%2BTcmiOc6kgvPMZepNRDxFz1GoT784xUt8%2Fuca%2FXL9B5n2iT2T6uAC40Pb8afuO6Rkn0qrhlKATUsLTr22VMNjyoF6sxdJjlv%2BWl2R0Au%2FpZhF9uMbjS0bPf7kx06zy7%2BTYRvZCnRFZlr0ISutWR2yDKYLL7%2FL%2FM48wmwNr2UmpCyR8qZ51%2BV6LW7Et%2FKv%2BzXcOTNGUZOnGHmyoi%2F2Z5gxhXvlDCxZGh17hweW4cFZTo4YmQnD75RJ8Py%2BT5m9PcWcVQ%2FuVqHMTNoV%2BuD2Y6xnM6UWNiH3yrUouXIESrGgnp9l2FnOWEDC6x8CpBjqdAWwKATmktHbBOLR2hYWD1lWXVFZb%2FlN%2Fo5AiK7WJ2PZ2pjnwonVYWk8%2BToNg1AXOeAzaDbNnT8NqYZaC5DyV1Z3djHY9OO2QsgxHrTCHp5Vi%2BKBFFRB4YnNHfef8C9oCPSsBSpFqjLviw66jr9MhkuqtBMU3ozXUQT0DtM%2FWAQxuoUU%2BV2Jta6zn1q1UelzJq2lb6JKRG9BCRWw8tAg%3D&X-Amz-SignedHeaders=host&x-id=GetObject&X-Amz-Signature=ccf5617b60e9c4dc89bc747fd567d2722cb1340ad5f3e13f90b4453f70068a7f) where that happened.) I also found I wasn't able to re-record these tests locally, either - maybe this is due to an SDK difference between my machine and CircleCI, but when I recorded locally, the play button in the A/V element was different, and the tests failed again.

I think we could remove _both_ remote elements here, and re-record all the tests, but I'm not sure I'll be able to fix them without digging further into those strange play button issues.